src/users.test.ts

```diff
@@ -64,9 +65,32 @@ describe('#createUser', () => {
     expect(newUser.data.username).toMatch(/^test/)
     expect(newUser.data.last_name).toEqual('Buddy')
     expect(newUser.data.custom_attrs).toEqual(custom_attrs)
-    expect(newUser.data.phone_numbers[0].number).toEqual('111-435-1334')
```

Comment:
Was this test failing for you?  It started failing for me after a git pull a little while ago, but I hadn't dug in yet

Reply:
oh i meant to comment on this. yes, it was failing for me.

when i inspected `newUser.data`, i didn't see anything related to phone numbers at all.